bfcfg

```diff
@@ -37,19 +37,27 @@ bfcfg_rc=0
 cfg_file=/etc/bf.cfg
 log_file=/tmp/bfcfg.log
 dump_mode=0
+verbose_mode=0
 
 mlxmkcap=/lib/firmware/mellanox/boot/capsule/scripts/mlx-mkcap
 
 bfcf_acpi_table=/sys/firmware/acpi/tables/BFCF
 efivars=/sys/firmware/efi/efivars
 sys_cfg_sysfs=${efivars}/BfSysCfg-9c759c02-e5a3-45e4-acfc-c34a500628a6
 redfish_cfg_sysfs=${efivars}/BfRedfish-ce3e5882-6770-4d5e-aa45-90f909da2446
+redfish_state_cfg_sysfs=${efivars}/BfRedfishState-ce3e5882-6770-4d5e-aa45-90f909da2446
+sb_state_sysfs=${efivars}/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
+sb_setup_mode_sysfs=${efivars}/SetupMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_global_var_guid=8be4df61-93ca-11d2-aa0d-00e098032b8c
 efi_bfcfg_var_guid=487ff588-fb71-4b19-9100-ebe067aa1af0
 efi_vlan_var_guid=9e23d768-d2f3-4366-9fc3-3a7aba864374
 rshim_efi_mac_sysfs=${efivars}/RshimMacAddr-${efi_global_var_guid}
 pxe_dhcp_class_id_sysfs=${efivars}/DhcpClassId-${efi_global_var_guid}
 bf_bundle_version_sysfs=${efivars}/BfBundleVer-${efi_bfcfg_var_guid}
+sb_custom_mode_sysfs=${efivars}/BfCfgSbCustomMode-${efi_bfcfg_var_guid}
+pass_settings_sysfs=${efivars}/BfCfgPassSettings-${efi_bfcfg_var_guid}
+passwd_sysfs=${efivars}/BfCfgPassword-${efi_bfcfg_var_guid}
+bf_modes_sysfs=${efivars}/BfCfgBfModes-${efi_bfcfg_var_guid}
```

Comment:
Hey Jim, I ended up removing the legacy password dump. I couldn't reproduce a scenario where the legacy password wasn't all 0s - the PwdCredential variable was either 0 or did not exist (which is similar to the secure password variable when no password is set). If it's correct that the legacy password is only used on older builds (<= 4.5) and gets converted to a secure password on newer builds then I think it's okay to just dump the secure password since the related edk2 changes won't be available on older builds.

Let me know what you think, happy to investigate more and if you have a suggested method to repro a non-zero legacy password in 4.8 I'll try that out. Thanks!

Reply:
Thats fine Chris. See no value including the Legacy Password. It only applies downgrading to an older (<= 4.5.x) release., otherwise as you mentioned it won't exist or be all 0's. Only reason I mentioned at all was because the wrong UEFI variable was being used to print it out.

Reply:
Sounds good, and yeah thanks for bringing that up!